src/CommunityToolkit.Maui.Core/Views/DrawingView/Service/DrawingViewService.macios.cs

```diff
@@ -15,14 +15,19 @@ public static class DrawingViewService
 	/// <param name="lines">Drawing lines</param>
 	/// <param name="imageSize">Maximum image size. The image will be resized proportionally.</param>
 	/// <param name="background">Image background</param>
+	/// <param name="token"><see cref="CancellationToken"/></param>
 	/// <returns>Image stream</returns>
-	public static ValueTask<Stream> GetImageStream(in IList<IDrawingLine> lines, in Size imageSize, in Paint? background)
+	public static ValueTask<Stream> GetImageStream(IList<IDrawingLine> lines, Size imageSize, Paint? background, CancellationToken token)
```

Comment:
One thing that I would do is to make a default value for all the `token` parameters. 

```suggestion
	public static ValueTask<Stream> GetImageStream(IList<IDrawingLine> lines, Size imageSize, Paint? background, CancellationToken token = default)
```

Reply:
Personally, I think it's best that Libraries do **not** provide a default value to `CancellationToken` for two reasons:
1. [Analyzer CA2016 ("Forward the 'cancellationToken' parameter methods that take one")](https://learn.microsoft.com/dotnet/fundamentals/code-analysis/quality-rules/ca2016) correctly alerts developers using our library to propagate their CancellationToken to our APIs
2. It forces the developer to make conscious decision about whether they want to pass in a cancellable Token  or not. Sometimes using `CancellationToken.None` is fine

That being said, I'm happy to disagree and commit here. I understand that most MAUI developers won't use `CancellationToken` and this extra parameter is just forcing them to type more code.

Reply:
All the BCL APIs provide a default value for CancellationToken, not sure about the reason behind it

Reply:
Ah - good find! 

Ok - I'll add default value to `CancellationToken` 👍